core/storage/user/gae_models.py

```diff
@@ -1904,6 +1904,12 @@ class UserQueryModel(base_models.BaseModel):
     edited_at_least_n_exps = datastore_services.IntegerProperty(default=None)
     # Query option to check if user has edited fewer than n explorations.
     edited_fewer_than_n_exps = datastore_services.IntegerProperty(default=None)
+    # Query option to check if user has created collection.
+    created_collection = datastore_services.StringProperty(default='None')
```

Comment:
Would it be possible to refactor the code to use `None, False, True` instead of strings with those?

Reply:
@vojtechjelinek  Instead should I just simplify and have just true or false, where false indicates the predicate not being considered(and not user not creating a collection)?

Reply:
Yes, that would be fine (and I assume even better than what you have now).

Reply:
Done.